assets/scss/settings/vendor/apple/_settings.scss

```diff
@@ -0,0 +1,12 @@
+// Apple
+// -----------------------------------------------------------------------------
+//
+// Purpose: Settings for Apple vendor styles
+//
+// 1. The color variables for the Apple Pay button are decoupled from
+//      Stencil's styles, to follow Apple's brand requirements.
+//
+// -----------------------------------------------------------------------------
+
+$applePay-black: #000000;
+$applePay-white: #FFFFFF;
```

Comment:
Please also add `// 1` after both of these variable declarations to reference the comment at the top